### PR TITLE
Add docker file for postgres 9.6 and 10

### PR DESCRIPTION
**Pull request by orhankislal**
- install plpython support for postgres
- add dockerfile for postgres 10 centos 7
- add postgres bin dir to $PATH for both 9.6 and 10
- remove unnecessary files

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/326/

File: tool/docker/base/Dockerfile_postgres_9_6 [hunk 1]
```diff
@@ -1,71 +0,0 @@
...
-
-### Install doxygen-1.8.13:
-RUN wget http://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.13.src.tar.gz && \
-      tar xf doxygen-1.8.13.src.tar.gz && \
-      cd doxygen-1.8.13 && \
-      mkdir build && \
-      cd build && \
-      cmake -G "Unix Makefiles" .. && \
-      make && \
-      make install
```
Comment (njayaram2) [inline tool/docker/base/Dockerfile_postgres_9_6]: We should probably keep this in the new docker files. 
These are required for `make doc` on the docker image. It is a convenient way of building docs for developers.

File: tool/docker/base/Dockerfile_postgres_9_6 [hunk 2]
```diff
@@ -1,71 +0,0 @@
...
-                       openssl \
-                       libssl-dev \
-                       libboost-all-dev \
-                       m4 \
-                       wget \
-                       vim \
-                       pgxnclient \
-                       flex \
-                       bison \
-                       graphviz
```
Comment (njayaram2) [inline tool/docker/base/Dockerfile_postgres_9_6]: We should keep `flex`, `bison`, and `graphviz` in `Dockerfile_centos_7_postgres_9_6` to help developers build docs using the docker image.

File: tool/docker/base/Dockerfile_postgres_9_6 [hunk 3]
```diff
@@ -1,71 +0,0 @@
...
-      cd build && \
-      cmake -G "Unix Makefiles" .. && \
-      make && \
-      make install
-
-### Optional: install LaTex
-### uncomment the following 'RUN apt-get' line to bake LaTex into the image
-### Note: if you run the following line, please tag the image as
-### madlib/postgres_9.6:LaTex, and don't tag it as latest
-# RUN apt-get install -y texlive-full
```
Comment (njayaram2) [inline tool/docker/base/Dockerfile_postgres_9_6]: Please retain this comment too in the new docker files. `texlive-full` is big to have it as part of the image, so developers can refer to this to figure out the exact command to install latex. It is required to build the design doc.

**Comment by orhankislal** [comment]
Created a new pull request (#228). Closing this one.

**Pull request closed without merging**